portfolio/src/main/webapp/index.html

```diff
@@ -105,16 +105,23 @@ <h2>Projects</h2>
                 </tr>
                 <tr>
                     <td>TEEN FAQ</td>
-                    <td></td>
                 </tr>
                 <tr>
                     <td>
-                        •	Created a website using HTML to help the transition into teenage years 
-                        •	Advised teenagers on topics ranging from schoolwork to health to fashion
-                        Checkout my website below!
+                        <p>•	Created a website using HTML to help the transition into teenage years </p>
+                        <p>•	Advised teenagers on topics ranging from schoolwork to health to fashion</p>
+                        <p>Checkout my website below!</p>
                     </td>
                 </tr>
-                <!-- need to add picture with link to website-->
+            </table>
+            <table>
```

Comment:
Table based layouts work but they're not really the best way to do things anymore. I'd look at CSS flexbox for your overall layout. There's a pretty fun way to learn what flex can do here: https://flexboxfroggy.com/

Reply:
Thank you! I will look into it